pkgs/by-name/fe/fermi2/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  config,
+  lib,
+  fetchFromGitHub,
+  zlib,
+  stdenv,
+}:
+stdenv.mkDerivation {
+  pname = "fermi2";
+  version = "0.1-unstable-2021-05-21";
+  src = fetchFromGitHub {
+    owner = "lh3";
+    repo = "fermi2";
+    rev = "cb1410972b2bd330883823116931ae67ead8b30f";
+    hash = "sha256-jDn1OBuGWDMEHI5A3R9meOykEGM6yjItSnUpx36DxgA=";
+  };
+  buildInputs = [ zlib ];
+  installPhase = ''
+    runHook preInstall
+
+    install -Dm755 fermi2 -t $out/bin
+
+    runHook postInstall
+  '';
+  meta = {
```

Comment:
Please mark as broken for aarch64-linux or fix
```
gcc -c -g -Wall -O2 -Wno-unused-function    ksw.c -o ksw.o
ksw.c:28:10: fatal error: emmintrin.h: No such file or directory
   28 | #include <emmintrin.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:13: ksw.o] Error 1
```

Reply:
Same here: adding back `    platforms = lib.intersectLists lib.platforms.x86_64 lib.platforms.unix;`